tests/chart_tests/test_houston_configmap.py

```diff
@@ -50,6 +50,24 @@ def test_houston_configmap():
     # Ensure elasticsearch client param is at the correct location and contains http://
     assert "node" in prod["elasticsearch"]["client"]
     assert prod["elasticsearch"]["client"]["node"].startswith("http://")
+
+    # Assert that the configMap contains required component tags
+    assert prod["deployments"]["helm"]["airflow"]["images"]["statsd"]["tag"] == "0.27.2"
+    assert prod["deployments"]["helm"]["airflow"]["images"]["redis"]["tag"] == "7.2.6"
+    assert prod["deployments"]["helm"]["airflow"]["images"]["pgbouncer"]["tag"] == "1.23.1-1"
+    assert prod["deployments"]["helm"]["airflow"]["images"]["pgbouncerExporter"]["tag"] == "0.18.0"
+    assert prod["deployments"]["helm"]["airflow"]["images"]["gitSync"]["tag"] == "4.2.3-1"
```

Comment:
We should not make assertions about specific tag version because we do not want to update the tests when the tag changes. The only exception is if we customized the tag to a test value, and are asserting against that customization.

Maybe we could change this to assert that the image is a valid semver, or matches `^[0-9]+\.[0-9]+\.[0-9]+\`, or is not `latest` or something like that?

Reply:
Personally, I'd remove all of these assertions about the tag value. I don't think that it's valuable enough to assert anything here. We don't make assertions about tag values anywhere else, and I don't think it's something we should start doing.